align/pdk/finfet/resistor.py

```diff
@@ -24,4 +25,7 @@ def generate(self, ports, netlist_parameters=None, layout_parameters=None):
         t = {'layer': 'Boundary', 'netName': None, 'rect': bbox}
         self.terminals.append(t)
 
+        # Additional metadata for layout post-processing
```

Comment:
This should be part of test & not functional code I think?

Reply:
I did not understand your comment. Could you please elaborate?

Reply:
This metadata, for example, could be used to instantiate PCELLs after PnR flow is completed.

Reply:
Meant to point to line 29
```python
self.metadata['instances'].append({'sample_key': 'sample_value'})
```

Reply:
I am assuming sample_key, sample_value are being used for testing. It isn't actual metadata you wish to export.

Reply:
Yes, that is correct, this is for testing purposes only.